tsl/src/continuous_aggs/materialize.c

```diff
@@ -272,6 +272,11 @@ spi_delete_materializations(SchemaAndName materialization_table, const NameData
 
 	if (res < 0)
 		elog(ERROR, "could not delete old values from materialization table");
+	else
+		elog(LOG,
```

Comment:
Nit: `row%s` is elegant. However, the processing of the logfile becomes more difficult and could be more error-prone (e.g., a regex to analyze the logfile needs to include both cases and a simple grep for `rows from materialization table` might miss the 1-row case).

Reply:
I expected that comment, but do you have a suggestion of a different formulation that does not require the plural "s"?

Reply:
Unfortunately not. I have only the uglier expressions `row(s)` / `tuple(s)` in my mind. However, PostgreSQL seems to follow this pattern:

https://github.com/postgres/postgres/blob/00f2a2556c9b23050e4f987e5d563b6856b83fc2/contrib/spi/refint.c#L600

Reply:
Followed that convention.